### ISSUE TITLE
more info for concatenated modules

### ISSUE THREAD

**Issue by sokra** [webpack-contrib/webpack-bundle-analyzer#157]
#### Issue description
Currently concatenated modules are displayed in a bit in-transparent way.
webpack 4 does include more information about these modules in stats, i. e. all modules which are included in the concatenated module.
With these information bundle analyser can display more details for concatenated modules. I would propose to display included modules as children in the same way as modules are currently displayed.
You only have the original size information, but you could guess the remaining sizes from the original module size factor.

``` text
    [0] ./parsing/harmony/node_modules/commonjs.js 91 bytes {0} [depth 1] [built]
    [1] ./parsing/harmony/node_modules/commonjs-trans.js 124 bytes {0} [depth 1] [built]
    [2] ./parsing/harmony/index.js + 10 modules 4.02 KiB {0} [depth 0] [built]
        | ./parsing/harmony/node_modules/circularEven.js 141 bytes [depth 1] [built]
        | ./parsing/harmony/index.js 3.11 KiB [depth 0] [built]
        | ./parsing/harmony/node_modules/reexport.js 110 bytes [depth 1] [built]
        | ./parsing/harmony/node_modules/reexport2.js 35 bytes [depth 1] [built]
        | ./parsing/harmony/node_modules/abc.js 175 bytes [depth 1] [built]
        | ./parsing/harmony/node_modules/exportKinds.js 133 bytes [depth 1] [built]
        | ./parsing/harmony/node_modules/exports-specifier.js 53 bytes [depth 1] [built]
        | ./parsing/harmony/node_modules/def.js 21 bytes [depth 1] [built]
        | ./parsing/harmony/node_modules/unused.js 110 bytes [depth 1] [built]
        | ./parsing/harmony/node_modules/abc_c.js 21 bytes [depth 2] [built]
        | ./parsing/harmony/node_modules/circularOdd.js 118 bytes [depth 2] [built]
```

#### Technical info
* Webpack version: 4

**Comment by th0r** [comment]
Fixed in `v2.11.0`

**Comment by sokra** [comment]
awesome
